internal/controller/pkg/revision/deployment.go

```diff
@@ -28,9 +28,10 @@ import (
 )
 
 var (
-	replicas                 = int32(1)
-	runAsUser                = int64(2000)
-	runAsGroup               = int64(2000)
+	replicas = int32(1)
+	// IBM Patch: Disable to run with restricted-scc in OpenShift
```

Comment:
should we comment it out? Or just remove the lines completely and then write a good commit message to describe the changes, e.g.:

> removed "runAsUser" and "runAsGroup" variables from deployment, so can run with restricted-scc in OpenShift

Reply:
I wanted to be able to quickly identify which files are modified by IBM by searching for `IBM Patch`

Reply:
sounds reasonable. For situations where upstream decides to remove those lines, we should remove them as well when syncing the fork.